driver/docker/docker_integration_test.go

```diff
@@ -0,0 +1,36 @@
+package docker
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/deislabs/cnab-go/driver"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriver_Run(t *testing.T) {
+	op := &driver.Operation{
+		Installation: "example",
+		Action:       "install",
+		Image:        "pvtlmc/example-outputs@sha256:ae3c1c719d87e981ecfdf84862ff389ce6a1281c99372f45d158f52e3d5be3dc",
```

Comment:
Could we build this bundle locally? I'm a little bit worried to run test against an unknown image pushed on a registry. If the image is accidentally deleted the test will fail and we won't be able to re-push the exact same image.
And most of all I think the whole sources of this integration test acts also as an example/documentation, and then without the `cnab/run` sources and/or the `Dockerfile` to produce the image, we are missing something.

Reply:
We ended up putting the bundle in our [duffle PR 815](https://github.com/deislabs/duffle/pull/815) under `examples/outputs` since thats where the other test bundles live.

Reply:
Ok just checked that, but from a strict dependency point of view, I think cnab-go shouldn't depend from duffle, even indirectly like here. Is it possible to move the `example/outputs` here in cnab-go and reverse the dependency?

Reply:
That makes sense to me, I can make that update to both PRs

Reply:
I saw you added the bundle.json and the Dockerfile needed to be able to rebuild the bundle, but the test is still depending on an image pushed on the Docker Hub. Do you think we can build the image locally before running the test (or maybe directly inside the test, invoking docker?) ?

Reply:
@silvin-lubecki just added a build script for this. Let me know what you think!